php/Terminus/Models/Environment.php

```diff
@@ -399,6 +399,56 @@ public function hasDeployableCode() {
   }
 
   /**
+   * Imports a full-site archive
+   *
+   * @param string $url URL to import data from
+   * @return Workflow
+   */
+  public function import($url) {
+    $params = [
+      'url'      => $url,
+      'code'     => 1,
+      'database' => 1,
+      'files'    => 1,
+      'updatedb' => 1,
+    ];
+
+    $workflow = $this->site->workflows->create(
+      'do_import',
+      ['environment' => $this->get('id'), 'params' => $params,]
+    );
+    return $workflow;
+  }
```

Comment:
This function needs to remain on the `Site` model because it involves code and therefore overwrites the entire site.